src/state/predictions/index.ts

```diff
@@ -59,6 +60,21 @@ export const fetchRoundBet = createAsyncThunk<
   return { account, roundId, bet: null }
 })
 
+/**
+ * Used to poll the user bets of the current round cards
+ */
+export const fetchCurrentBets = createAsyncThunk<
+  { account: string; bets: Bet[] },
+  { account: string; roundIds: string[] }
+>('predictions/fetchCurrentBets', async ({ account, roundIds }) => {
+  const betResponses = await getBetHistory({
+    user: account.toLowerCase(),
+    round_in: roundIds,
```

Comment:
I realized that we can query the current round's bets more efficiently by using the `round_in` clause. Before each card was querying its bet individually.